the bluez lib doesn't expose any symbols for some reason, so manually do it for the used one
diff --git a/simplebluez/src/Device.cpp b/simplebluez/src/Device.cpp
index d00badb..086a11a 100644
--- a/simplebluez/src/Device.cpp
+++ b/simplebluez/src/Device.cpp
@@ -50,7 +50,7 @@ std::shared_ptr<Characteristic> Device::get_characteristic(const std::string& se
     return service->get_characteristic(characteristic_uuid);
 }
 
-void Device::pair() { device1()->Pair(); }
+__attribute__ ((visibility ("default"))) void Device::pair() { device1()->Pair(); }
 
 void Device::cancel_pairing() { device1()->CancelPairing(); }
 
